conviva/src/main/java/com/bitmovin/analytics/conviva/MetadataOverrides.java

```diff
@@ -14,6 +14,7 @@ public class MetadataOverrides {
     private String applicationName;
     private Map<String, String> custom;
     private Integer duration;
+    private Map<String, Object> additionalStandardTags;
```

Comment:
Shouldn't this be `Map<String, String>`? Does Conviva even support us sending objects?